src/main/java/org/osiam/auth/exception/OsiamExceptionHandler.java

```diff
@@ -60,6 +60,14 @@ protected JsonErrorResult handleResourceNotFound(HttpServletRequest request,
         return new JsonErrorResult(HttpStatus.NOT_FOUND.name(), e.getMessage());
     }
 
+    @ExceptionHandler(ClientAlreadyExistsException.class)
+    @ResponseStatus(HttpStatus.CONFLICT)
+    @ResponseBody
+    protected JsonErrorResult handleClientAlreadyExists(ClientAlreadyExistsException e) {
+        LOGGER.warn("A ClientAlreadyExistsException occurred", e);
```

Comment:
Yes, only the already [in the controller created exception message](https://github.com/dacrome/auth-server/blob/handle-duplicate-client-creation/src/main/java/org/osiam/auth/oauth_client/ClientManagementController.java#L75) should be logged. But I think it should be logged on info level instead of warn, right?